client/me/account/main.jsx

```diff
@@ -373,7 +373,7 @@ const Account = React.createClass( {
 					href={ config( 'signup_url' ) }
 					onClick={ this.recordClickEvent( 'Primary Site Add New WordPress Button' ) }
```

Comment:
@apeatling do you know if changing the event name would mess up historical data in GA?